cmd/cloud-node-manager/app/config/config.go

```diff
@@ -81,4 +81,9 @@ type Config struct {
 	// WindowsService should be set to true if cloud-node-manager is running as a service on Windows.
 	// Its corresponding flag only gets registered in Windows builds
 	WindowsService bool
+
+	// DeprecatedApplyBetaTopologyLabels indicates whether the node should apply beta topology labels.
+	// If true, the node will apply beta topology labels.
+	// DEPRECATED: This flag will be removed in a future release.
+	DeprecatedApplyBetaTopologyLabels bool
```

Comment:
I don't think we should change the behavior for the versions that have already been released, it may break some existing v1.26+ clusters.

Could we change this to something like "EnableDepecatedBetaTopologyLabels" and default it to false?

Reply:
> I don't think we should change the behavior for the versions that have already been released, it may break some existing v1.26+ clusters.

Agreed. The current behaviour is that the behaviour stays the same as master now, unless you opt in, ie the flag defaults to false and we only apply the labels when the flag is explicitly specified

> Could we change this to something like "EnableDepecatedBetaTopologyLabels" and default it to false?

I can go through and rename everything, sure